deployments/aws/lb-connectors/terraform.tfvars.sample

```diff
@@ -117,6 +113,9 @@ customer_master_key_id = "<key-id>"
 #    3. unicode characters
 # See: https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements
 
+# If using encryption, enter the CMK ID. Otherwise, comment out.
+# customer_master_key_id = "<key-id-uuid>"
```

Comment:
The section above under For option 1 (plaintext), it says already to comment out customer_master_key_id. Maybe for here, we can say uncomment if using encryption?

Reply:
ok